.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+	"presets": [
+		["@babel/preset-env", {
+			"targets": {
+				"chrome": "74",
+				"firefox": "67"
+			}
+		}]
+	]
+}
```

Comment:
As Sindre mentioned, Babel can probably be dropped altogether.

Reply:
Can you comment on https://github.com/sindresorhus/notifier-for-github/pull/180#issuecomment-501557959. I'd like some feedback on that comment.